app/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "name-examination",
-  "version": "1.2.32",
+  "version": "1.2.37",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "name-examination",
-      "version": "1.2.32",
+      "version": "1.2.36",
```

Comment:
Shouldn't this match line 3?

Did you create this lock file using `pnpm i`? You should not manually change this file.